Chap_Introduction.tex

```diff
@@ -143,71 +74,121 @@ \section{PMIx Architecture Overview}
+
+As clients invoke \ac{PMIx} \acp{API}, it is possible that some client requests can 
+be handled at the client level.  Other requests might require communication with the 
+local \ac{PMIx} server, subsequently might request services from the host \ac{SMS} 
+(represented here by a \ac{RM} daemon).  
+The interaction between the \ac{PMIx} server and \ac{SMS} are 
+achieved using callback functions registered during server initialization.  
+The host \ac{SMS} can indicate its lack of support for any operation by simply providing a \textit{NULL} for the associated callback function, or can create a function entry that returns \textit{not supported} when called.
 
 Recognizing the burden this places on SMS vendors, the PMIx community has included interfaces by
-which the host can request support from local SMS elements. Once the SMS has transferred the request to
-an appropriate location, a PMIx server interface can be used to pass the request between SMS subsystems.
+which the host can request support from local SMS elements. Once the \ac{SMS} has transferred the request to
+an appropriate location, a \ac{PMIx} server interface can be used to pass the request 
+between \ac{SMS} subsystems.
 For example, a request for network traffic statistics can utilize the
-PMIx networking abstractions to retrieve the information from the FM. This reduces the portability and
+PMIx networking abstractions to retrieve the information from the Fabric Manager. 
+This reduces the portability and
 interoperability issues between the individual subsystems by transferring the burden of defining the
-interoperable interfaces from the SMS subsystems to the PMIx community, which continues
+interoperable interfaces from the \ac{SMS} subsystems to the \ac{PMIx} community, which continues
 to work with those providers to develop the necessary support.
 
-Tools, whether standalone or embedded in job scripts, are an exception to the communication rule and can connect to
-any PMIx server providing they are given adequate rendezvous information. The PMIx conceptual model views the
-collection of PMIx servers as a cloud-like conglomerate --- i.e., orchestration and information requests can be
-given to any server regardless of location. However, tools frequently execute on locations that may not house an
-operating PMIx server --- e.g., a users notebook computer. Thus, tools need the ability to remotely connect to
-the PMIx server ``cloud''.
+Fig.~\ref{fig:roles} shows how tools can interact with the \ac{PMIx} architecture.
+Tools, whether standalone or embedded in job scripts, are an exception to 
+the normal client registration process.   A process can register as a tool, provided 
+the \ac{PMIx} client library has adequate rendezvous information to connect to the appropriate
+\ac{PMIx} server.  This allows processes
+which were not created by the PMIx infrastructure to request access to PMIx functionality. 
+Processes registered as tools do not have peers.  
```

Comment:
Per discussion on teleconf - strike this last sentence. It _may_ be possible that tools have peers (see `PMIX_TOOL_NSPACE` / `PMIX_TOOL_RANK` in the `PMIx_tool_init` call)